language-bindings/python/wasm-c-api/README.md

```diff
@@ -0,0 +1,7 @@
+# WASM-C-API
```

Comment:
Had better not upload the binary file `sun.wasm`, it can already been generated by compile.sh.